.github/workflows/Release.yml

```diff
@@ -39,72 +39,12 @@ jobs:
           draft: true
           prerelease: false
 
-      - name: Upload Crane
+      - name: Upload app
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: Crane/app/build/outputs/apk/debug/app-debug.apk
-          asset_name: crane-debug.apk
+          asset_path: app/build/outputs/apk/demo/debug/app-demo-debug.apk
+          asset_name: app-demo-debug.apk
```

Comment:
If we want to do a release this way, don't we want to create the release version of the app? APKs we want to distribute shouldn't have the performance implications of the debug build.

Reply:
+1, but how is the demoRelease apk signed?

Reply:
As discussed, it is currently signed with the debug keystore

Reply:
Updated to release the demo release apk!